trl/trainer/grpo_trainer.py

```diff
@@ -536,6 +543,36 @@ def new_group_context():
             # desynchronization and seems to lead to DeepSpeed hanging during initialization. To prevent this, we
             # synchronize all processes after vLLM has been fully initialized.
             self.accelerator.wait_for_everyone()
+        elif self.use_remote_vllm:
+            if self.accelerator.is_main_process:
+                from trl.extras.remote_vllm_helper import VllmRemoteClient, get_remote_vllm_script_file
+
+                assert args.vllm_max_model_len is not None, "vllm_max_model_len must be set when using remote vllm"
+                # import pdb
+                remote_gpus = ",".join([str(i) for i in range(args.remote_vllm_num_gpus)])
```

Comment:
this will as the user to set the CUDA_VISIBLE_DEVICES=`0,1` for example when `remote_vllm_num_gpus=2` should it not be the last 2 as per the example in the `grpo_with_remote_vllm.py`?

Reply:
Yes, there CUDA_VISIBLE_DEVICES should be the GPUs NOT used by the accelerate training processes. 
But I do not know how to recommend which GPUs 😂, as the VLLM may not be running in same machine, 
there are too many available options.